src/models/gan/train.py

```diff
@@ -0,0 +1,95 @@
+import tensorflow as tf
+import data.process as preprocess
+import os
+import matplotlib.pyplot as plt
+
+from models.common.training import Trainer
+from data.nsynth import nsynth_from_tfrecord, instruments, nsynth_to_melspec
+from models.gan.model import GAN
+
+#import IPython.display as display
+
+# Some compatability options for some graphics cards
+from tensorflow.compat.v1 import ConfigProto
+from tensorflow.compat.v1 import InteractiveSession
+import tensorflow_datasets as tfds
+
+config = ConfigProto()
+config.gpu_options.allow_growth = True
+session = InteractiveSession(config=config)
```

Comment:
Perhaps this should be in the training script rather than in each model?

Reply:
Done